app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
```

Comment:
- [ ] OPTIONAL
It is a nice idea to use one of the Rails Active Callbacks `after_save` to call the `update_post_likes_counter method every time a new like has been saved to database, then make the `update_post_likes_counter` method private, similar to:

```ruby
 # app/models/likes.rb
  after_save :update_like_counter

  private

  def update_like_counter
    author.increment!(:likes_counter)
  end
```

Reply:
Thanks